### ISSUE TITLE
Nexus: Traktor 3.11.0 Crashes When Connecting Kontrol S5

### ISSUE THREAD

**Issue by skymakai** [ErikMinekus/traktor-kontrol-screens#28]
As described in Title, Traktor 3.11.0 Crashes when connecting Kontrol S5 using the Nexus mod.

**Comment by ErikMinekus** [comment]
Tested with the D2 and it works fine...

I think the only file that has changed specifically for the S5 is [CSI/S5/S5Deck.qml](https://github.com/ErikMinekus/traktor-kontrol-screens/commit/e031687a877516485dc2b8d63dbe98919b16e47b#diff-598a96126fef7338735185f1a331323c9f473eda0ea1e1050847e0bfbe0fa817). Are you sure you replaced all files?

**Comment by skymakai** [comment]
Yes, absolutely sure all files have been replaced.  CSI, Defines, Screens.
Downloaded each of the 3 mods earlier today and tested each one after the other for compatibility, each time completely replacing the CSI, Defines, and Screens folders with yours.

Found key matching not working with Main and Prime, but the immediate crash with Nexus (doesn't happen unless the Kontrol S5 is connected).

**Comment by ErikMinekus** [comment]
If it's still an issue and you're on a Mac, could you run Traktor from Terminal with the following command:
`/Applications/Native\ Instruments/Traktor\ Pro\ 3/Traktor.app/Contents/MacOS/Traktor`

And post the output here? That should print out why it's crashing.

**Comment by andresblack** [comment]
Same here. Windows 10 and S5. Traktor 3.11. Main and nexus works just fine but Prime shut down tracktor

**Comment by lsmith77** [comment]
> Same here. Windows 10 and S5. Traktor 3.11. Main and nexus works just fine but Prime shut down tracktor

FTR the issue was about Nexus and not Prime or was there a typo in your comment?
